domains/pallets/messenger/src/extensions.rs

```diff
@@ -402,6 +495,7 @@ where
             // prepare if this is a messenger call and has been validated
             (Some(messenger_call), Val::ValidatedRelayMessage(validated_relay_message)) => {
                 MessengerExtension::<Runtime>::do_prepare(messenger_call, validated_relay_message)
+                    .map(|_| ())
```

Comment:
This is not consistent with `MessengerExtension`. `MessengerTrustedMmrExtension` is used in FP and invalid bundle detection, the only difference between `MessengerExtension` and `MessengerTrustedMmrExtension` should be that the latter does not verify the MMR proof, besides that it should return the same weight and refund the same amount of weight and also have the `post_dispatch_details` implemented.